Rakefile

```diff
@@ -3,4 +3,8 @@ require 'rubocop/rake_task'
 
 RuboCop::RakeTask.new
 
-task default: %w(rubocop)
+require 'scraper_test'
+ScraperTest::RakeTask.new.install_tasks
+
+task test: 'test:data'
+task default: %w(rubocop test)
```

Comment:
this looks like you're running the old version of rubocop here. Is the plan to update that separately?